tools/install_ssl_configuration.sh

```diff
@@ -0,0 +1,320 @@
+SCRIPT_PATH="${BASH_SOURCE[0]}"
+if ! [ -x "$(command -v realpath)" ]; then
+    if [ -L "$SCRIPT_PATH" ]; then
+
+        [ -x "$(command -v readlink)" ] || { echo "Not able to resolve symlink. Install realpath or readlink.";exit 1; }
+
+        # try readlink (-f not needed because we know its a symlink)
+        DIR="$( cd "$( dirname $(readlink "$SCRIPT_PATH") )" && pwd -P)"
+    else
+        DIR="$( cd "$( dirname "$SCRIPT_PATH" )" && pwd -P)"
+    fi
+else
+    DIR="$( cd "$( dirname "$(realpath "$SCRIPT_PATH")" )" && pwd -P)"
+fi
+
+echo "OpenSearch Security Demo Installer"
+echo " ** Warning: Do not use on production or public reachable systems **"
+
+OPTIND=1
+assumeyes=0
+initsecurity=0
+cluster_mode=0
+skip_updates=-1
+use_defaults=0
```

Comment:
`use_default` seems redundant. `assumeyes` does same/similar thing. Can we reuse that?